Link libcxxabi with gcc_eh when linking with gcc_s is
not desired. Normally, llvm's libunwind would be used but
we are not using libunwind in ChromeOS right now because of
some reported crashes when libunwind was used.
https://crbug.com/747113

diff --git a/src/CMakeLists.txt b/src/CMakeLists.txt
index efbea67..e39335f 100644
--- a/src/CMakeLists.txt
+++ b/src/CMakeLists.txt
@@ -68,8 +68,10 @@ if (LIBCXXABI_USE_LLVM_UNWINDER)
   else()
     list(APPEND libraries unwind)
   endif()
+elseif(LIBCXXABI_HAS_GCC_S_LIB)
+    list(APPEND libraries gcc_s)
 else()
-  append_if(libraries LIBCXXABI_HAS_GCC_S_LIB gcc_s)
+    list(APPEND libraries gcc_eh)
 endif()
 
 # Setup flags.
